src/sentry/static/sentry/app/views/organizationContext.jsx

```diff
@@ -42,6 +42,7 @@ const OrganizationContext = createReactClass({
     useLastOrganization: PropTypes.bool,
     organizationsLoading: PropTypes.bool,
     organizations: PropTypes.arrayOf(SentryTypes.Organization),
+    detailed: PropTypes.number,
```

Comment:
We should make this a boolean since that's what it is -- we'll have to translate it to 0/1 in the API request. Otherwise, imagine a consumer of this component trying to use it, they won't know it should be a binary value.